xbmc/addons/binary/interfaces/api1/PVR/AddonCallbacksPVR.cpp

```diff
@@ -365,6 +418,9 @@ typedef struct EpgEventStateChange
 
 void CAddonCallbacksPVR::UpdateEpgEvent(const EpgEventStateChange &ch, bool bQueued)
 {
+  if (!m_enabled)
+    return;
+
```

Comment:
The interface is not protected by a lock and can be disabled while in PVREpgEventStateChange. So better to check here too.